evap/results/exporters.py

```diff
@@ -71,11 +71,12 @@ def init_grade_styles(cls):
                 "This can happen, if the file is imported / run multiple "
                 "times in one application run.",
                 ImportWarning,
+                stacklevel=2,
```

Comment:
I guess this triggered [no-explicit-stacklevel (B028)](https://docs.astral.sh/ruff/rules/no-explicit-stacklevel/).

I'm a bit confused about this rule. The rule description (linked above) says:
> The warnings.warn method uses a stacklevel of 1 by default, which limits the rendered stack trace to that of the line on which the warn method is called.
> 
> It's recommended to use a stacklevel of 2 or higher, give the caller more context about the warning.

which sounds like "this is the backtrace depth that you will get printed out", while [the documentation of warnings.warn](https://docs.python.org/3/library/warnings.html#warnings.warn) says:
> The stacklevel argument can be used by wrapper functions written in Python, like this:
> 
> ```
> def deprecated_api(message):
>   warnings.warn(message, DeprecationWarning, stacklevel=2)
> ```
> This makes the warning refer to deprecated_api’s caller, rather than to the source of deprecated_api itself (since the latter would defeat the purpose of the warning message).

which sounds to me as if "the warning trigger point will be reported as if the warning was raised N levels higher in the stacktrace".